package.json

```diff
@@ -63,7 +63,7 @@
     "@storybook/react": "^6.5.16",
     "@storybook/testing-library": "0.0.13",
     "@storybook/theming": "^6.5.16",
-    "@testing-library/jest-dom": "^5.10.1",
+    "@testing-library/jest-dom": "^5.16.5",
```

Comment:
out of curiosity, what happens if you uninstall this in the root?

Reply:
uninstall `@testing-library/jest-dom` entirely on the root? `npm ci` fails

![Screenshot 2023-03-16 at 4 20 27 PM](https://user-images.githubusercontent.com/104797704/225743369-dd26fcaa-04a5-4abe-8972-61e8c27f122f.png)

Reply:
hmm. weird. i still don't get it tho 😆 anyway, this looks gooooood